js/everything.js

```diff
@@ -211,6 +217,30 @@ var ZenPlayer = {
                 that.show();
                 updateTweetMessage();
             });
+            // when player has finished playing
+            plyrPlayer.addEventListener("ended", function() {
+                videoMetaData = getParsedVideoMetaData();
+                if (videoMetaData) {
```

Comment:
What if this video is already in `items`? We should update it instead of adding a duplicate entry